app/services/vot/supported_component_values.rb

```diff
@@ -34,7 +34,7 @@ module SupportedComponentValues
     ).freeze
     Pb = ComponentValue.new(
       name: 'Pb',
-      description: 'A biometric comparison is required as part of identity proofing',
+      description: 'IAL2 features are enabled (experimental)',
       implied_component_values: ['P1'],
       requirements: [:biometric_comparison],
```

Comment:
The thread mentioned "two pieces of fair evidence", etc, do we need to update requirements?

I feel like the `description` is not worth sweating that much, since what really matters is the documentation on the developer site?

Reply:
We have not added the 2 pieces of fair evidence feature yet. Once that is added we will update the requirements here.